locators/main_page_locators.py

```diff
@@ -21,3 +21,52 @@ class GoogleMainPageLocators:
     BOTTOM_PRIVACY = (By.XPATH, "//body/div[@id='viewport']/div[@id='main']/div[@id='footer']/div[@id='fbarcnt']/div[@id='footcnt']/div[@id='fbar']/div[1]/span[1]/a[1]")
     BOTTOM_TERMS = (By.XPATH, "//body/div[@id='viewport']/div[@id='main']/div[@id='footer']/div[@id='fbarcnt']/div[@id='footcnt']/div[@id='fbar']/div[1]/span[1]/a[2]")
     BOTTOM_SETTINGS = (By.XPATH, "//a[@id='fsettl']")
```

Comment:
Need to delete class GoogleMainPageLocators! We don't use already Google page for testing